eng/Versions.props

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <!-- The .NET Core product branding version -->
     <ProductVersion>5.0.0</ProductVersion>
+    <AssemblyVersion>5.0.0.0</AssemblyVersion>
```

Comment:
What's our plan for servicing?

Reply:
I will create an issue to track the work required for 5.0 shipping.